### ISSUE TITLE
Problem with less

### ISSUE THREAD

**Issue by githubjeka** [almasaeed2010/AdminLTE#736]
My less:
```less
@import "../vendor/bower/bootstrap/less/bootstrap.less";
@import "admin-lte/AdminLTE.less";
```

Form inputs have `@gray` color for text after compile

![image](https://cloud.githubusercontent.com/assets/874234/10479425/f4ffcd68-726c-11e5-899a-a9d40503acb5.png)

Because:
In Bootsrap `@gray:    lighten(@gray-base, 33.5%); // #555`
In admin-lte `@gray:    #d2d6de;`

**Comment by ivantcholakov** [comment]
Related: #274

**Comment by almasaeed2010** [comment]
Hello @githubjeka 

I didn't change the variable name because we have the issue of creating errors in customized AdminLTE versions. Developers who use a customized AdminLTE version will get inconsistent colors. This issue will be fixed in AdminLTE 3.0. We apologize for the inconvenience!
